popups/fave-tab/popup.js

```diff
@@ -0,0 +1,16 @@
+export default async ({ addon, msg, safeMsg }) => {
+  window.vue = new Vue({
+    el: "body",
+    data: {
+      projects: [],
+      projectsVisable: false,
+    },
+    async created() {
+      const username = await addon.auth.fetchUsername();
+      const res = await fetch(`https://api.scratch.mit.edu/users/${username}/favorites/?limit=40`);
```

Comment:
Is there a way to fetch more than 40 projects? I want to add a search bar to and be able to search all favorites.

Reply:
I believe that there's a `page` parameter for fetching the next set of 40, but I haven't checked if that's implemented for favorites.

Reply:
There's an `offset` parameter that offsets individual items, so you can set it to multiples of 40 to show different pages.